functions/get_puppet_servers.pp

```diff
@@ -0,0 +1,11 @@
+function rsan::get_puppet_servers() {
+  $puppet_servers =
```

Comment:
$puppet_servers needs to be in a format 

puppet_metrics_dashboard::master_list expects